modules/core/src/views/view.js

```diff
@@ -56,6 +59,17 @@ export default class View {
     Object.seal(this);
   }
 
+  get controller() {
+    return this._controller;
+  }
+
+  set controller(value) {
+    if (typeof value === 'function') {
+      value = {type: value};
+    }
+    this._controller = value ? Object.assign({}, this.constructor.defaultController, value) : null;
```

Comment:
Still not convinced that static definition is helpful. I think we could just have the View subclass set the controller in its constructor or return it through a "virtual" method.